packages/user-interface/style.css

```diff
@@ -2,6 +2,10 @@
   pointer-events: none;
 }
 
+.octotree-show .octolinker-toast  {
+  left: var(--octotree-sidebar-width) !important
```

Comment:
It feels wrong that we have to really on class names and variables added by another browser extension, but this is the best solution I was able to come up with. I tried to move the toast notification into another diff so it gets pushed with the entire content, but this didn't worked out.

Reply:
Adjusting the z index wasn't enough?

Reply:
Do be honest, I haven't really thought about that 😆  I can give it another try this weekend. Thanks for the hint